### PR TITLE
update safe sysctls

### PR DESCRIPTION
**Pull request by hunshcn**
https://github.com/kubernetes/kubernetes/blob/v1.27.0/pkg/kubelet/sysctl/safe_sysctls.go#L39
https://github.com/kubernetes/kubernetes/blob/v1.27.0/staging/src/k8s.io/pod-security-admission/policy/check_sysctls.go#L80

### CODE REVIEW

**Comment by Ritikaa96** [comment]
please make seperate changes for each localisation and recommit this PR with only English localisation changes

**Comment by atoato88** [comment]
/unassign 
As mentioned by previous comments, please create PR in English content first.
I'm reviewer in Japanese content, so removed my assignment.

**Comment by hunshcn** [comment]
@Ritikaa96 done

**Comment by Ritikaa96** [comment]
Hi @hunshcn , Highly Appreciate your contribution here.
Would you please add some description in the title so that the changes are understood.?

**Comment by hunshcn** [comment]
> Hi @hunshcn , Highly Appreciate your contribution here. Would you please add some description in the title so that the changes are understood.?

of course, changed. Should I alse change the commit message ?

**Comment by sftim** [comment]
@kubernetes/sig-node-pr-reviews does this change look correct?

**Review by endocrimes — APPROVED** [review]
/lgtm

File: content/en/docs/concepts/security/pod-security-standards.md
Comment (hunshcn) [reply content/en/docs/concepts/security/pod-security-standards.md:274]: This is what I deliberately ignored because of the discussion [here](https://github.com/kubernetes/kubernetes/pull/115374/files#r1135895707).

Comment (endocrimes) [reply content/en/docs/concepts/security/pod-security-standards.md:274]: 👍  sgtm

**Comment by hunshcn** [comment]
pinging @endocrimes  for approval

**Comment by pacoxu** [comment]
/lgtm
https://github.com/kubernetes/kubernetes/pull/115374

https://github.com/kubernetes/kubernetes/pull/118846 added another in v1.29.

**Comment by hunshcn** [comment]
It seems to be all right. Is there anything else to do?

**Comment by Ritikaa96** [comment]
> It seems to be all right. Is there anything else to do?

LGTM.
You have to wait for the approval before this PR get merge.

**Comment by Ritikaa96** [comment]
Please read [Pull Request Process](https://www.kubernetes.dev/docs/guide/pull-requests/) for further information regarding contribution processes.

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm

**Comment by hunshcn** [comment]
pinging @tengqm  for approval

**Pull request merged**
